src/viztracer/report_builder.py

```diff
@@ -67,18 +68,28 @@ def load_jsons(self) -> None:
                 self.jsons = [get_json(self.data)]
             elif isinstance(self.data, (list, tuple)):
                 self.jsons = []
+                self.invalid_json_paths = []
                 for idx, j in enumerate(self.data):
                     if self.verbose > 0:
                         same_line_print(f"Loading trace data from processes {idx}/{len(self.data)}")
-                    self.jsons.append(get_json(j))
+                    try:
+                        self.jsons.append(get_json(j))
+                    except json.JSONDecodeError:
+                        self.invalid_json_paths.append(j)
+                if len(self.invalid_json_paths) > 0:
+                    self.final_messages.append(("corrupted_json", {"paths": self.invalid_json_paths}))
 
     def combine_json(self) -> None:
         if self.verbose > 0:
             same_line_print("Combining trace data")
         if self.combined_json:
             return
-        if not self.jsons:
+        if not self.jsons and not self.invalid_json_paths:
             raise ValueError("Can't get report of nothing")
+        elif not self.jsons:
+            # print corrupted json files
+            self.print_messages()
+            raise ValueError("All json files are corrupted")
```

Comment:
```suggestion
        if not self.jsons:
            if not self.invalid_json_paths:
                raise ValueError("Can't get report of nothing")
            else:
                raise ValueError("No valid json files found")
```

Reply:
Changing the exception message should be enough - this is a rather rare case, just separate that from "no files".